src/Avalonia.Controls/TopLevel.cs

```diff
@@ -232,6 +235,18 @@ protected bool AutoSizing
         }
 
         /// <inheritdoc/>
+        IRenderTarget IRenderRoot.CreateRenderTarget()
+        {
+            return _renderInterface.CreateRenderer(PlatformImpl.Handle);
```

Comment:
Ok, sounds good - though I think that should be a separate PR; I'd like to keep this focused on the render loop change. As I said it's intended to be part of a much larger change. As such does it look good to merge now to you? If so could you merge? I'd like to use the system where the reviewer merges the PR rather than the author.